src/impl/apis/funds_api.py

```diff
@@ -1,34 +1,49 @@
 # coding: utf-8
 
-from typing import List
+from typing import List, Optional
 import logging
+import uuid
 
+from fastapi import HTTPException
 from fastapi_utils.cbv import cbv
 from spec.apis.funds_api import FundsApiSpec, router as FundsApiRouter
```

Comment:
FundsApiRouter -> fundsApiRouter, and please do the same with other imports of router( lowecase as lowercase) and please change them where they are used too.